corehq/apps/fixtures/templates/fixtures/csql_fixture_configuration.html

```diff
@@ -0,0 +1,71 @@
+{% extends 'hqwebapp/bootstrap5/base_section.html' %}
+{% load hq_shared_tags %}
+{% load i18n %}
+
+{% js_entry_b3 "hqwebapp/js/htmx_and_alpine" %}
+
+{% block page_title %}
+  {% trans 'CSQL Fixture Configuration' %}
+{% endblock %}
+
+{% block page_content %}
+  <div x-data="{
+                validationFailed: false,
+                isDirty: false,
+                rows: {{ csql_fixture_configurations|safe }},
+                addRow() {
+                    this.rows.push({ row: '', csql: '', id: '' });
+                },
+                deleteRow(index) {
+                    this.rows.splice(index, 1);
+                    this.markDirty();
+                },
+                markDirty() {
+                    this.isDirty = true;
+                },
+                unmarkDirty() {
+                    this.isDirty = false;
+                }
+            }">
+    <form hx-post={{ save_url }} action="/submit" method="POST" hx-target="#response-message" hx-swap="innerHTML"
+        @change="markDirty" @submit="unmarkDirty">
+      <div id="response-message"></div>
+      <button type="submit" class="btn btn-primary" id="Save" x-bind:disabled="!isDirty">{% trans "Save" %}</button>
```

Comment:
What would you think of handling the save and delete actions for each row individually (with a save button per row)?  Seems like it'd make things a bit more straightforward, while still being usable.  Plus then the "delete" action would take place right away.

Reply:
Cool, maybe after this is all merged I'll take a look at it more closely - I think I'll need to try it out locally to really understand.  Been meaning to find an excuse to try out alpine and HTMX.  Maybe a mob coding thing during a capacity building session?

Reply:
Yes please.